### PR TITLE
Jakarta Persistence 3.0 Draft

### PR DESCRIPTION
**Pull request by lukasj**
## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat-spec-x.y.pdf
- [x] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Progress Review, that sufficient progress has been made on
      a Compatible Implementation and TCK, to ensure that the spec is
      implementable and testable.
- [ ] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://repo1.maven.org/maven2/jakarta/persistence/jakarta.persistence-api/3.0.0-RC2/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      https://download.eclipse.org/ee4j/jakartaee-tck/
- [x] The URL of the compatibility certification request issue:
      N/A

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

File: persistence/3.0/_index.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ---
 title: "Jakarta Persistence 3.0 (under development)"
```
Comment (kwsutter) [inline persistence/3.0/_index.md:2]: Let's change "(under development)" to "(milestone)" in the header to clarify what's being delivered.  This is consistent with the PRs that have come before you.  Thanks!

File: persistence/3.0/_index.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 * [Jakarta Persistence 3.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
-  * [jakarta.persistence:jakarta.persistence-api:jar:3.0.0]()
+  * [jakarta.persistence:jakarta.persistence-api:jar:3.0.0-RC2](https://search.maven.org/artifact/jakarta.persistence/jakarta.persistence-api/3.0.0-RC2/jar)
 
 
 # Compatible Implementations
 
-* [Jakarta Persistence]()
+* [EclipseLink 3.0.0-M2](https://www.eclipse.org/eclipselink)
```
Comment (kwsutter) [inline persistence/3.0/_index.md:21]: Should this point to somewhere more specific?  For example, I can't find an M2 driver for EclipseLink via the link you provided.  I went to Downloads and couldn't find it.  So, I tried Milestone downloads and still didn't find it.  I then went to the 3.0 web page (https://www.eclipse.org/eclipselink/releases/3.0.php) and found a link to Download 3.0, but that takes me back this page, which only has 2.7.7 drivers on it (https://www.eclipse.org/eclipselink/downloads/index.php#27).

File: persistence/3.0/_index.md (hunk 1 above)
Comment (lukasj) [reply persistence/3.0/_index.md]: done

File: persistence/3.0/_index.md (hunk 2 above)
Comment (lukasj) [reply persistence/3.0/_index.md]: idea was to create PRs and update web for each project as we get close to the GF M1 date. But by not updating the web one can save some time, so links were updated to what we have right now and will be updated eventually if new milestones become available

**Pull request merged**